yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -178,18 +293,42 @@ func prepareImportDataStatusTable() ([]*tableMigStatusOutputRow, error) {
 			status = "MIGRATING"
 		}
 		row := &tableMigStatusOutputRow{
-			fileName:           path.Base(dataFile.FilePath),
-			tableName:          dataFile.TableName,
-			status:             status,
-			totalCount:         totalCount,
-			importedCount:      importedCount,
-			percentageComplete: perc,
+			tableName:  dataFile.TableName,
+			schemaName: getTargetSchemaName(dataFile.TableName),
+		}
+		if withStreamingMode {
+			qualifiedTableName := qualifyTableName(tconf.Schema, row.tableName)
+			eventCounter, err := tdb.GetImportedEventsStatsForTable(qualifiedTableName, migrationUUID)
+			if err != nil {
+				return nil, fmt.Errorf("get imported events stats for table %q: %w", qualifiedTableName, err)
+			}
+			row.totalEvents = eventCounter.TotalEvents
+			row.numInserts = eventCounter.NumInserts
+			row.numUpdates = eventCounter.NumUpdates
+			row.numDeletes = eventCounter.NumDeletes
+			row.importedCount, err = state.GetImportedRowCount(dataFile.FilePath, dataFile.TableName)
+			if err != nil {
+				return nil, fmt.Errorf("get imported row count for table %q: %w", qualifiedTableName, err)
+			}
+			row.finalRowCount = row.importedCount + row.numInserts - row.numDeletes
+			if (isffDB && checkfallForwardComplete()) || (!isffDB && checkCutoverCompleted()) { //TODO: handle for fallforward-complete
+				row.status = "DONE"
+			} else {
+				row.status = "STREAMING"
+			}
+		} else {
+			row.fileName = path.Base(dataFile.FilePath)
+			row.status = status
+			row.totalCount = totalCount
+			row.importedCount = importedCount
+			row.percentageComplete = perc // TODO: for streaming mode, this should be the percentage of events processed.
```

Comment:
Don't we need to populate these fields for both snapshot and wihtStreaming modes? Consider restoring them in the initialisation stage as they were earlier.

Reply:
what about the percentage in streaming?